packages/core/src/textures/TextureSystem.js

```diff
@@ -81,8 +73,6 @@ export default class TextureSystem extends System
 
         gl.bindTexture(gl.TEXTURE_CUBE_MAP, this.emptyTextures[gl.TEXTURE_CUBE_MAP].texture);
 
-        let i;
```

Comment:
Rather than removing this, you should move it into the intialization portion of the for loop.